src/core/system.js

```diff
@@ -57,6 +49,28 @@ System.prototype = {
   init: function () { /* no-op */ },
 
   /**
+   * Update handler. Called during scene attribute updates.
+   * Systems can use this to dynamically update their state.
+   */
+  update: function (oldData) { /* no-op */ },
+
+  updateProperties: function (rawData) {
```

Comment:
The `updateProperties` method of a [component](https://github.com/aframevr/aframe/blob/master/src/core/component.js#L232) calls `update`. Should we do the same for consistency?